packages/mobile/src/components/header-button/header-button.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC } from 'react';
+import { BackIcon, Pressable, Text } from '~/components/components';
+import { globalStyles } from '~/styles/styles';
+import { styles } from './styles';
+
+type Props = {
+  label: string;
+  onPress: () => void;
+};
+
+const HeaderButton: FC<Props> = ({ label, onPress }) => {
+  return (
+    <Pressable
+      onPress={onPress}
+      style={[
+        globalStyles.flexDirectionRow,
+        globalStyles.alignItemsCenter,
+        {
+          width: '100%',
```

Comment:
Why do we need to specify width explicitly here?

Also let's move `justinfyContent-` items to globalStyles, e.g.:
```
justinfyContentStart: {
    justinfyContent: 'flex-start',
},
justinfyContentBetween: {
    justinfyContent: 'space-between',
},
...
```

Reply:
corrected